src/player.js

```diff
@@ -0,0 +1,57 @@
+class Player {
+  constructor(col, row, img) {
+    this.col = col;
+    this.row = row;
+    this.width = PLAYER_WIDTH;
+    this.height = PLAYER_HEIGHT;
+    this.img = img;
+  }
+
+  preLoad() {
+    this.img = loadImage("./assets/character-down.png");
+  }
+
+  drawPlayer() {
+    image(this.img, this.row, this.col, this.width, this.height);
+  }
+
+  moveUp() {
+    if (keyDown(ARROW_UP)) {
```

Comment:
You can remove the if statements from all these `move__` methods, since they are not needed here. Instead, create a `KeyPressed` function in `main.js` and in the `Game` & `Player` classes.

```main.js
function keyPressed() {
  game.keyPressed();
}
```